apps/cms/lib/partial/paragraph/code_embed.ex

```diff
@@ -0,0 +1,30 @@
+defmodule CMS.Partial.Paragraph.CodeEmbed do
+  @moduledoc """
+  Represents a CodeEmbed Paragraph in the CMS.
+  Body is raw HTML from CMS, and is not scrubbed or rewritten.
+  """
+
+  alias CMS.Partial.Paragraph.ColumnMultiHeader
+  alias Phoenix.HTML
+
+  import CMS.Helpers, only: [field_value: 2, parse_paragraphs: 3]
+
+  defstruct header: nil,
+            right_rail: false,
+            body: HTML.raw("")
+
+  @type t :: %__MODULE__{
+          header: ColumnMultiHeader.t() | nil,
```

Comment:
`header` is so this embedded code can be "introduced" with custom HTML text, so that this block can be easily moved around the page w/o losing context. This is the same behavior as content lists and multi column paragraphs. These "blocks" basically have an intro/context/title chunk, and the actual content chunk. Together it makes a modular block.

Reply:
Is this needed for the 2 embeds we have currently? Given that the whole point of this paragraph type is explicitly to not work like other paragraphs and be "outside the flow", I can see an argument for keeping it as simple as possible.